text/0000-plugin-package-format.md

```diff
@@ -0,0 +1,79 @@
+# Summary
+A unified way for plugins to be packaged for all platforms.
+
+# Motivation
+Currently all platforms have their own way of shipping platforms, some even being different by the way they are installed (system-wide vs local-user), which results in higher difficulty when shipping cross-platform plugins. The current behavior also prevents some much needed future features from even working correctly.
+
+# Drawbacks
+None.
+
+# Design
+Plugins should be shipped and loaded in this format on all platforms from now on:
+
+- `/<name>/data`
+    The plugins custom data, such as localized text and images.
+- `/<name>/bin/<platform>/<arch>-<bits>/<name>.<ext>`
```

Comment:
Consider making `<arch>-<bits>` optional, in order to support alternative ways to ship plugins (Lua, Python, multi-arch).